WordPress/Classes/ViewRelated/Me/AccountSelectionHelper.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  AccountSelectionView.swift
+//  WordPress
+//
+//  Created by Gonzalo G Erro on 11/8/16.
+//  Copyright © 2016 WordPress. All rights reserved.
+//
+
+import Foundation
+
+struct Account {
+    var userId: NSNumber
+    var username: String
+    var email: String
+
+    init(userId: NSNumber, username: String, email: String) {
+        self.userId = userId
+        self.username = username
+        self.email = email
+    }
+}
+
+protocol AccountSelectionHelperDelegate: class {
+    func selectedAccount(account: Account)
+}
+
+class AccountSelectionHelper: NSObject {
+
+    var accountsListView: UIView?
+    var titleView: UIView
+    weak var delegate: AccountSelectionHelperDelegate?
+    let parentView: UIView
+    var accounts: [Account]
+    let height: CGFloat = 44.0
+
+    var tableViewController = UITableViewController()
+    var handler: ImmuTableViewHandler!
+
+    let chevronImageView: UIImageView = { () -> UIImageView in
+        let chevron = UIImage.init(named: "theme-type-chevron")
+        return UIImageView.init(image: chevron)
+    }()
+
+    let disclosureChevronImageView: UIImageView = {
+        let chevron = UIImage.init(named: "disclosure-chevron")
+        return UIImageView.init(image: chevron)
+    }()
+
+    var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text =  NSLocalizedString("Me", comment: "Me page title")
+        label.font = UIFont.init(name: "Helvetica-Bold", size: 17)
```

Comment:
This is not the font we'd typically use for that label, see `-[WordPressAppDelegate customizeAppearance]`.